.vscode/extensions.json

```diff
@@ -0,0 +1,15 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
```

Comment:
What about including `aaron-bond.better-comments`? We have a lot of comments in the code, and I think this extension helps a lot to make them more visible...

Reply:
I want to keep all nice to have things that are purely aesthetically to each user, as some may not want colors on certain things that causes them to stand out etc.

But if you want it in we can add it as well